index.js

```diff
@@ -12,11 +12,10 @@ module.exports = {
   // ],
 
   locals(options) {
-    let name = options.entity.name;
-    let component = componentize(name);
+    let name = componentize(options.entity.name);
```

Comment:
And then we need to update `files/config/environment.js` to use `name` instead of `component` for the `modulePrefix`